src/PSRule.Rules.Azure/rules/Azure.MariaDB.Rule.ps1

```diff
@@ -30,6 +30,12 @@ Rule 'Azure.MariaDB.DefenderCloud' -Ref 'AZR-000330' -Type 'Microsoft.DBforMaria
     }
 }
 
+# Synopsis: Azure Database for MariaDB servers should reject TLS versions older than 1.2.
+Rule 'Azure.MariaDB.MinTLS' -Ref 'AZR-000335' -Type 'Microsoft.DBforMariaDB/servers' -Tag @{ release = 'GA'; ruleSet = '2022_12'; } {
+    $Assert.HasFieldValue($TargetObject, 'properties.minimalTlsVersion', 'TLS1_2').
+    Reason($LocalizedData.MinTLSVersion, $TargetObject.properties.minimalTlsVersion)
```

Comment:
@bengeset96 Thought of something else. Where getting the reason `Minimum TLS version is set to {0}.` because the property `$TargetObject.properties.minimalTlsVersion` is null.

Lets just remove `.Reason($LocalizedData.MinTLSVersion, $TargetObject.properties.minimalTlsVersion)`. I don't think we need a custom reason here the default reason should be fine, and it already handles this scenario.

Reply:
@BernieWhite 

Yes, I was also thinking that, but saw it has been used for other rules previously. I'll change it and perhaps we also can change the other rules with the same pattern.